DZSwiftUI/DZSwiftUI/ContentView.swift

```diff
@@ -7,22 +7,179 @@
 
 import SwiftUI
 
-///
+/// Экран настройки
 struct ContentView: View {
+    
+    private enum Constants {
+        static let onOffPickerValue = ["Выкл.", "Вкл."]
+        static let onOffPickerBluetoothSectionValue = 0
+        static let onOffPickerModemSectionValue = 0
+        static let avatarImageCornerRadius = 40
+        static let avatarImageFrameWidth = 80
+        static let avatarImageFrameHeight = 80
+        static let avatarImageName = "yinAndYange"
+        static let userName = "Alexander"
+        static let userOfferText = "Apple ID, iCloud, контент и покупки"
+        static let userOfferLineLimit = 1
+        static let userOfferFontSize = 11
+        static let arrowImageName = "chevron.right"
+        static let arrowImageFrameWidth = 7
+        static let arrowImageFrameHeight = 11
+        static let settingsOfferTextOne = "Предложения Apple ID"
+        static let notificationNumberOne = "3"
+        static let notificationNumberTwo = "1"
+        static let notificationCircleRadius = 25
+        static let navigationBarTitle = "Настройки"
+        static let settingsOfferTextTwo = "iOS 16.2 уже доступно"
+        static let settingsImageCornerRadius = 5
+        static let forEachFirstValue = 0
+        static let settingsAirplaneImageName = "avia"
+        static let settingsAirplaneNameText = "Авиарежим"
+        static let settingsWiFiImageName = "wifi"
+        static let settingsWiFiNameText = "Wi-Fi"
+        static let settingsWiFiText = "Не подключено"
+        static let settingsBluetoothImageName = "bluetooth"
+        static let settingsBluetoothNameText = "Bluetooth"
+        static let settingsRelationsImageName = "relations"
+        static let settingsRelationsNameText = "Сотовая связь"
+        static let settingsModemImageName = "modem"
+        static let settingsModemNameText = "Режим модема"
+        static let settingsVPNImageName = "vpn"
+        static let settingsVPNNameText = "VPN"
+    }
+    
+    @State private var isOnAirPlaneToggle = false
```

Comment:
```suggestion
    @State private var isOnAirPlaneToggle = false
```
все private Ниже var body